frontend/src/component/section/Details.tsx

```
@@ -0,0 +1,138 @@
+                          className="block dark:text-white text-gray-900 font-bold text-2xl mb-2 hover:text-indigo-600 transition duration-500 ease-in-out"
+                        >
+                          {a && a.title}
+                        </Link>
+                        <p className="text-gray-700  text-base mt-2 mx-5 sm:mx-10">
+                          {/* {a && truncateText(a.content, 100)} */}
+                        </p>
+                      </div>
+                    </div>
+                  </div>
+                </>
+              ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)